source/likely.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
 var likely = require('./index.js');
+var history = require('./history');
+var utils = require('./utils');
 
 window.likely = likely;
-window.addEventListener('load', likely.initiate);
+window.addEventListener('load', function () {
+    if (likely.defaults.listenChangesUrl) {
+        history.onChangesUrl = function () {
```

Comment:
@vitkarpov, oh, wait. So we initialize the history API only when Likely is being loaded from the browser. Is this intended?